### ISSUE TITLE
Please Add Support for The Rehub Theme To Over ride all woocommerce Style Functions 

### ISSUE THREAD

**Issue by Dilstroyer** [pojome/elementor#6359]
## Prerequisites

- [x] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate. 
- [x] The feature is still missing in the latest stable version of Elementor ( Elementor Pro. )

**What problem is your feature request going to solve? Please describe.**
Hi,
This is my first time posting as I just went pro.

After Elementor premium support looked into my case and inspected my theme, it appears that my theme Rehub by Sizam is restricting woocommerce styling options, namely the **product archive widget for styling add to cart, buttons , variations, typology and actually most woocommerce styling functions.** 

I have reached out to the developer but they dont seem too interested in making the theme fully open to support Elementor right now. I wanted to change themes but I have allot of custom code built in to the child theme that has cost me a few hundred $ and the thought of having to do a redo is causing me  major headaches.. 

**Describe the solution you'd like**

I would really appreciate it if Elementor's developers are able to test & override this themes woocommerce template. According to the developer this file holds the key so I'm hoping the pros at Elementor code HQ can unlock the E.M product archive widget.

**Describe alternatives you've considered**
I have asked on a few facebook community groups but the best suggestion was coding it with css which I have done a little of, but it is a huge time thief and not a long term solution, as the woo builder was the reason I went pro..

**Additional context**
So all I know is Rehub has a customisation.php file that can be overridden but I have no clue how to do this and have tried several times without success. 

I have included the theme files and the customization .php file for the developers to look at.  
[rehub.zip](https://github.com/pojome/elementor/files/2651748/rehub.zip)
[REhub-customization.zip](https://github.com/pojome/elementor/files/2651753/REhub-customization.zip)

Big thanks in advance !

**Comment by bainternet** [comment]
@Dils

After checking your theme, it seems like it loads the theme customization at the `wp_head` hook so you can easily disable that for product pages and archives via a simple snippet in your theme's functions.php file:

```PHP
add_action( 'wp_head', function() {
	if ( is_product() || is_product_taxonomy() || is_shop() ) {
		remove_action( 'wp_head', 'rehub_custom_css' );
	}
}, 9 );
```

**Comment by Dilstroyer** [comment]
Hi,

Thank you for looking into this. 

However, adding this code has no effect whatsoever in changing the **add to cart buttons colors, backgrounds, column parameters, variations or box in woo product archive widget** in elementor pro.. 
I suspect there may be allot more going on in the theme. 

I actually just discovered by trial and error, the **woocommerce.css in the Rehub css folder** holds all woo styling. 
Renaming the entire woocommerce folder is the only thing that sets the woo template to default but then I lose all my customizations. 
Note: I am using the child theme Re-vendor that has a grandchild custom plugin that has some custom css but not for the product archive so rehub main parent css applies. To test, I also deactivated that custom plugin and it still didn't work even after clearing all cache.

Checkout this file. 
[woocommerce.css.zip](https://github.com/pojome/elementor/files/2666509/woocommerce.css.zip)

Did you actually get it to work in a product archive with rehub and elementor pro just by dropping in that code ?

Thanks again for your assistance.
